Dockerfile

```diff
@@ -1,27 +1,30 @@
-FROM golang:1.19.7-alpine3.17 as builder
+FROM --platform=$BUILDPLATFORM golang:1.21.1-alpine3.18 as builder
 RUN apk add --no-cache gcc git make musl-dev
 
 COPY ./go.mod /app/go.mod
 COPY ./go.sum /app/go.sum
 COPY ./Makefile /app/Makefile
 
-COPY ./bindings /app/bindings
+COPY ./op-node /app/op-node
+COPY ./op-chain-ops /app/op-chain-ops
+COPY ./op-service /app/op-service
+COPY ./op-batcher /app/op-batcher
+COPY ./op-bindings /app/op-bindings
 COPY ./components /app/components
```

Comment:
`kroma-validator` is better

Reply:
Since this is a dependency on the makefile, any modification here will require the makefile build to run in two suits. How about renaming the validator directory to kroma-validator and moving the components level up?